rocketpy/environment/environment.py

```diff
@@ -2737,6 +2743,96 @@ def decimal_degrees_to_arc_seconds(angle):
         arc_seconds = (remainder * 60 - arc_minutes) * 60
         return degrees, arc_minutes, arc_seconds
 
+    def to_dict(self, include_outputs=False):
+        env_dict = {
+            "gravity": self.gravity,
+            "date": self.date,
+            "latitude": self.latitude,
+            "longitude": self.longitude,
+            "elevation": self.elevation,
+            "datum": self.datum,
+            "timezone": self.timezone,
+            "_max_expected_height": self.max_expected_height,
```

Comment:
```suggestion
            "max_expected_height": self.max_expected_height,
```

Reply:
This is a blocker

Reply:
This is already solved. I forgot to resolve the conversation.